apps/meteor/tests/e2e/page-objects/omnichannel-livechat.ts

```diff
@@ -142,6 +142,13 @@ export class OmnichannelLiveChat {
 		return this.page.getByRole('link', { name: fileName });
 	}
 
+	public async openChatAndSendMessage(liveChatUser: { name: string; email: string }, message = '') {
+		await this.openLiveChat();
+		await this.sendMessage(liveChatUser, false);
+		await this.onlineAgentMessage.fill(message);
+		await this.btnSendMessageToOnlineAgent.click();
```

Comment:
I really don't like multi-level pageobjects, it diverts the focus of the test, it forces the tester to understand the test programming